Src/IronPython/Runtime/MemoryView.cs

```diff
@@ -342,11 +342,11 @@ public sealed class MemoryView : ICodeFormattable, IWeakReferenceable, IBufferPr
         public Bytes tobytes() {
             CheckBuffer();
 
-            if (_shape?.Contains(0) ?? false) {
+            if (_shape.Contains(0)) {
                 return Bytes.Empty;
             }
 
-            var buf = _buffer.AsReadOnlySpan();
+            var buf = _buffer!.AsReadOnlySpan();
```

Comment:
Is there a smart way to indicate `_buffer` being not null after the `CheckBuffer()` call, withouth changing the signature of `CheckBuffer()`?

Reply:
I believe [MemberNotNullAttribute](https://docs.microsoft.com/en-us/dotnet/api/system.diagnostics.codeanalysis.membernotnullattribute?view=net-5.0) would allow that, unfortunately it's still considered a preview feature.